### PR TITLE
Add lost container preloading info

### PR DESCRIPTION
**Pull request by kprosise**
Information that was on the removed
`reference-manual/linux/preloaded-images` page that did not migrate to the user-guide page was added.

QA: visually inspected html, checked with linter. No issues spotted.

No associated Ticket/Issue, as this was a quick fix.

# PR Template and Checklist

Please complete as much as possible to speed up the reviewing process.

Readiness and adding reviewers as appropriate is required.

All PRs should be reviewed by a technical writer/documentation team and a peer.
If effecting customers—which is a majority of content changes—a member of Customer Success must also review.

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

Focus was mostly upon copying the old information and cleaning it up. Open to suggestions if context or content needs additional work. Redirect was also added.

## Checklist

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 547245d90899072a5df01a0e4ba5289a9556693d are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2646/docs/artifacts/html/index.html

**Comment by mwasilew** [comment]
LGTM

**Review by mike-sul — APPROVED** [review]
Thanks!

File: source/user-guide/containers-and-docker/container-preloading.rst [hunk 1]
```diff
@@ -234,5 +234,82 @@ Verify the ``compose-apps-early-start`` application status:
 
 After the ``compose-apps-early-start`` service has been successfully run, ``docker ps`` will show that the preloaded apps are running.
 
+Common Advanced Scenario
+------------------------
+
+More complex workflows are common.
+For example, a Factory may have ``containers.git`` set up with multiple branches where each specifies a different set of apps.
+
+Assume you have four branches with the following application:
```
Comment (angolini) [inline source/user-guide/containers-and-docker/container-preloading.rst:243]: I assume the 4 branches are: devel, experimental, main, production

it took me 3 seconds to understand "money-making-app" is not the branch name

I'm only saying, I don't know if anything should be changed.

**Review by angolini — APPROVED** [review]

**Review by vanmaegima — APPROVED** [review]

**Review by MrCry0 — APPROVED** [review]

**Pull request merged**